### PR TITLE
Translate 2.2.5 and 2.3.1 release posts (vi)

### PR DESCRIPTION
**Pull request by nguyenducgiang**

### CODE REVIEW

File: vi/news/_posts/2016-04-26-ruby-2-3-1-released.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* [https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.1.zip](https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.1.zip)
+
+      SIZE:   19842037 bytes
+      SHA1:   ab9dec602b11ee4cfc295d6aa87ebe712372d123
+      SHA256: 4c8ae431b33f78d64cabb31911e0890e9a3ac380b4f22b11738f9baeeda51763
+      SHA512: a26d3ab5983c6f3ea454e3e75554137305525479e4c15c0ae424689e870e2c5a9f0fe194975cf362cc5528ce601e31a0a15b87c7af200fd0d1da17459435b953
+
+## Lời nhắn
+
+Xin gửi lời cảm ơn tới các commiters, các nhà phát triển, và những người dùng đã cung cấp thông báo lỗi, đã giúp chúng tôi có thể phát hành phiên bản này.
```
Comment (nhattan) [inline vi/news/_posts/2016-04-26-ruby-2-3-1-released.md]: ```committers``` thay vì ```commiters```
e nghĩ liệu có thể thay thế ```committers``` bằng ```các cộng tác viên``` được không nhỉ?

**Comment by stomar** [comment]
Please rewrap to a line length of less than 80 characters, and amend the commit, and force push your branch. Also the commit message should be changed to e.g. `Translate 2.2.5 and 2.3.1 release posts (vi)`.

File: vi/news/_posts/2016-04-26-ruby-2-3-1-released.md (hunk 1 above)
Comment (joneslee85) [inline vi/news/_posts/2016-04-26-ruby-2-3-1-released.md]: từ `cộng tác viên` nghe có vẻ hợp lý 👍

**Comment by nguyenducgiang** [comment]
@stomar @joneslee85 @nhattan Updated. Thanks.

**Comment by stomar** [comment]
Please follow the guidelines for commit messages next time; note that PR title and commit message are two different things.

**Pull request merged**